Cohere.Tests/CohereStepDefinitions.cs

```diff
@@ -0,0 +1,41 @@
+using Reqnroll;
+using Xunit;
+
+namespace Cohere.Tests;
+
+[Binding]
+
+/// <summary>
+/// Defines the step definitions used by all Cohere tests
+/// </summary>
+public class CohereStepDefinitions
+{
+    public CohereClient _client;
```

Comment:
ok. You can do partial class here.
For the other stepDefinitions, you can declare them as partial